ui/scss/src/_studio-view.scss

```diff
@@ -206,6 +206,7 @@
   }
 
   .site-dashboard {
```

Comment:
Above on line 157 (`#activeContentActions`), I see the selected content thingie is positioned with visibility hidden. This would overlay and block pointer events from what's below. Check that that isn't an issue now that the top margin is removed.